client/asset/dash/dash.go

```diff
@@ -0,0 +1,169 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package dash
+
+import (
+	"fmt"
+
+	"decred.org/dcrdex/client/asset"
+	"decred.org/dcrdex/client/asset/btc"
+	"decred.org/dcrdex/dex"
+	dexbtc "decred.org/dcrdex/dex/networks/btc"
+	dexdash "decred.org/dcrdex/dex/networks/dash"
+
+	"github.com/btcsuite/btcd/chaincfg"
+)
+
+const (
+	version                 = 0
+	BipID                   = 5      // 1 used for derivation on for testnet/regtest
+	minNetworkVersion       = 190100 // Dash v19.1.0, proto: 70227
+	walletTypeRPC           = "dashdRPC"
+	defaultRedeemConfTarget = 2
+)
+
+var (
+	configOpts = append(btc.RPCConfigOpts("Dash", "9998"), []*asset.ConfigOption{
```

Comment:
I think I know what you mean but if you could expand a bit that would help.

In general dash blocks can be considered final after 2 blocks.

Reply:
`redeemconftarget` is a target for now many blocks to wait unit a miner is likely to get it at all.  Generally this will be the argument to `estimatesmartfee`.  In dash, it's unlikely to matter at all since it's not a busy chain.  We just have to check that `estimatesmartfee 1` doesn't return screwy values, as is the case with some assets.

Reply:
I have implemented the `"redeemconftarget"` option with a `DefaultValue` of 2.
From my testing on Dash 1 is not an acceptable input to `estimatesmartfee`